rails-commenting.rb

```diff
@@ -48,15 +48,15 @@ def update
   def destroy
     @post = BlogPost.find(params[:id])
     if @post.destroy
-      # ---8)
+      # ---8)Redirects to the index of the blog post
       redirect_to blog_posts_path
     end
   end
 
-  # ---9)
+  # ---9)It allows only the paramaters we want into the model from a submission, protects our model from unwanted data
   private
   def blog_post_params
-    # ---10)
+    # ---10) Whats permitted for this blog post model is the title and the content
```

Comment:
great job overall.  Make sure you understand which pages the redirects are going to.  if there is a param passed in the path, then it's not the index. Otherwise - awesome work!